code/modules/surgery/organs/heart.dm

```diff
@@ -23,9 +23,8 @@
 			Stop()
 			return
 
-	spawn(120)
-		if(!owner)
-			Stop()
+	if(!special)
```

Comment:
If the organ is just straight up deleted from a mob or  something similar, it's pointless to have the timer checked for.